content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -35,18 +35,17 @@ you have selected.  Kubernetes recommends using KMS v2.
 
 * Kubernetes version 1.10.0 or later is required
 
+* For version 1.29 and later, the feature is disabled by default.
+  To enable the feature, set `--feature-gates=KMSv1=true` to configure a KMS v1 provider.
+
 * Your cluster must use etcd v3 or later
 
 ### KMS v2
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state for_k8s_version="v1.29" state="stable" >}}
 
 * For version 1.25 and 1.26, enabling the feature via kube-apiserver feature gate is required.
```

Comment:
```suggestion
```
The feature-state shortcode is a hint that the doc won't apply to older versions (or is certainly intended to be).